command/repo_test.go

```diff
@@ -296,12 +294,12 @@ func TestRepoFork_outside_survey_yes(t *testing.T) {
 	cs.Stub("") // git clone
 	cs.Stub("") // git remote add
 
-	oldConfirm := Confirm
-	Confirm = func(_ string, result *bool) error {
+	oldConfirm := prompt.Confirm
+	prompt.Confirm = func(_ string, result *bool) error {
```

Comment:
Wow I did not know that another package could override a value in the imported package 😲

Still, even if supported, this has a smell to it. Would it be a better idea to instead extend the `prompt` package with a function like `StubPrompt` that returns a teardown function?

Reply:
yes, i was just in a hurry ^_^()